scheduler/roles/slurm/tasks/cleanall.yml

```diff
@@ -0,0 +1,58 @@
+---
+# tasks file for slurm
+- name: Populate service facts
+  ansible.builtin.service_facts:
+
+- name: Stop Services
+  ansible.builtin.systemd_service:
+    name: "{{ item }}"
+    state: stopped
+  when: item in ansible_facts['services'].keys()
+  # ignore_errors: true
+  loop:
+    - slurmctld.service
+    - slurmdbd.service
+    - slurmd.service
```

Comment:
can we have it in vars?

Reply:
used only once, so no need, its better readable here